src/components/users/add-member-dialog.tsx

```diff
@@ -118,9 +121,11 @@ const AddMemberGroupDialog: React.FC<Props> = ({
         <Button disableElevation variant="contained" onClick={ onCancel } color="secondary" autoFocus>
           { strings.userManagementScreen.addMemberDialog.cancelButton }
         </Button>
-        <Button onClick={ onCreateClick } color="primary" disabled={ !valid }>
-          { strings.userManagementScreen.addMemberDialog.createButton }
-        </Button>
+        <GenericLoaderWrapper loading={ loading }>
```

Comment:
Maybe we could just disable the button when something is loading instead of using loader?